opentuner/search/composableevolutionarytechniques.py

```diff
@@ -359,19 +395,68 @@ def select_parameters(self, params):
     """
     randomly select a subset of parameters to operate on
     """
-    ret_list = []
     random.shuffle(params)
-    for i, k in enumerate(params):
-      if i < self.n_cross or random.random() < self.cr:
+    ret_list = params[:self.must_mutate_count]
+    for param in params[self.must_mutate_count:]:
+      if random.random() < self.cr:
         ret_list.append(k)
     return ret_list
 
   def get_default_operator(self, param_type):
     return {'op_name': 'op4_set_linear', 'args': [1.0, self.use_f, -self.use_f], 'kwargs': {}}
 
+class GreedyComposableTechnique(ComposableEvolutionaryTechnique):
+  """
+  Always mixes in global best as parents
+  """
+  def __init__(self,
+               mutation_rate = 0.1,
+               must_mutate_count = 1,
+               *pargs, **kwargs):
+    super(GreedyComposableTechnique, self).__init__(*pargs, **kwargs)
+    self.mutation_rate = mutation_rate
+    self.must_mutate_count = must_mutate_count
+    self.population_size = 10
```

Comment:
I wanted a lower population size for this, but just realized I can just kwarg it and override.